### ISSUE TITLE
Feature names get dropped if only one Feature in a Theme

### ISSUE THREAD

**Issue by edwardsmarc** [NCC-CNC/wheretowork#343]
I'm doing some data prep for a small AOI which results in a couple of the national data themes only having a single species.

When opened in the app, the theme name (e.g. **Critical Habitat: Threatened Species (ECCC)**) is displayed along with the color legend, but the species name (e.g. **Bank swallow**) is dropped.

![image](https://github.com/NCC-CNC/wheretowork/assets/10728298/0eb69e5d-2e5f-4e18-ae46-e48f8c33ed7a)

For cases where a Theme only has one Feature, I wonder if we can display them using the same UI code as Themes with multiple Features. So you'd have the nested structure with only a single layer in the Theme dropdown.

I realise this may have been setup for Themes like **Rivers** and **Shoreline** that will only ever have one Feature per Theme and so look nicer with the simplified display. But losing the species names is a problem. 

This could be fixed in data prep by changing the Theme name in the meta data table, but that get's a bit unruly because the name gets quite long (and I prefer the consistency of having all Themes display in the same style):

![image](https://github.com/NCC-CNC/wheretowork/assets/10728298/af141e18-97ba-4d16-afa5-87ca9ca67f74)

**Comment by DanWismer** [comment]
I agree, it would be nice if a single theme could have the nested structure. And also the the single look for features such as Rivers.

**Comment by ssast** [comment]
@edwardsmarc If still interested in handling this on the front end, I've got a solution to display the single-feature themes nested in the same way as the multi-feature themes e.g. below. The weights, includes, and excludes remain as is.

Current:
<img width="405" height="77" alt="Image" src="https://github.com/user-attachments/assets/e5d12ee4-38c1-44a3-8e64-02c050539e17" />

Revised:
<img width="401" height="114" alt="Image" src="https://github.com/user-attachments/assets/9a627694-9058-47b9-9295-cac8c88b982c" />

**Comment by edwardsmarc** [comment]
Thanks @ssast, I would vote to implement this!
